### DISCUSSION TITLE
`encore run` producing an FTL error.

### DISCUSSION THREAD

**Discussion by braswelljr** [encoredev/encore#580] [Help]
FTL connection failure error="rpc error: code = Unavailable desc = error reading from server: read unix `@->C:\\Users\\brasw\\AppData\\Local\\encore\\encored.sock`: wsarecv: An existing connection was forcibly closed by the remote host."

**Comment by eandre** [comment]
Which version of Encore are you running? Are you building it from source or running an official build?

**Comment by braswelljr** [comment]
I'm running the latest official build on Windows 11.
